test/test_helper.rb

```diff
@@ -1,16 +1,27 @@
 require 'simplecov'
 require 'simplecov-rcov'
+require 'codeclimate-test-reporter'
 require 'shoulda'
 if ENV['JENKINS'] == 'true'
   SimpleCov.formatter = SimpleCov::Formatter::RcovFormatter
+  true
+else
+  SimpleCov.formatter = SimpleCov::Formatter::MultiFormatter[
+      SimpleCov::Formatter::HTMLFormatter,
+      CodeClimate::TestReporter::Formatter
+  ]
 end
 
-require 'codeclimate-test-reporter'
-CodeClimate::TestReporter.start
+CodeClimate::TestReporter.configure do |config|
+  config.git_dir = "#{Dir.pwd}/plugins/redmine_issue_templates"
+end
 
-# FIXME: Remove 'rails' because same issue is happened when run test on CI environment.
-#    Ref. https://github.com/colszowka/simplecov/issues/82
-SimpleCov.start
+SimpleCov.start do
+  add_filter do |source_file|
+    # report this plugin only.
+    !source_file.filename.include?('plugins/redmine_issue_templates') || !source_file.filename.end_with?(".rb")
```

Comment:
__[rubocop]__
* Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)